diff --git a/.local/bin/statusbar/sb-battery b/.local/bin/statusbar/sb-battery
index 93cbe08..327af9e 100755
--- a/.local/bin/statusbar/sb-battery
+++ b/.local/bin/statusbar/sb-battery
@@ -10,9 +10,9 @@ case $BLOCK_BUTTON in
 🔌: charging
 ⚡: charged
 ❗: battery very low!
-- Scroll to change adjust xbacklight." ;;
-	4) xbacklight -inc 10 ;;
-	5) xbacklight -dec 10 ;;
+- Scroll to change adjust light." ;;
+	4) light -inc 10 ;;
+	5) light -dec 10 ;;
 	6) "$TERMINAL" -e "$EDITOR" "$0" ;;
 esac
 
diff --git a/.local/bin/sysact b/.local/bin/sysact
index 64c2f32..6d4b86d 100755
--- a/.local/bin/sysact
+++ b/.local/bin/sysact
@@ -2,10 +2,7 @@
 
 # A dmenu wrapper script for system functions.
 export WM="dwm"
-case "$(readlink -f /sbin/init)" in
-	*systemd*) ctl='systemctl' ;;
-	*) ctl='loginctl' ;;
-esac
+ctl='systemctl'
 
 wmpid(){ # This function is needed if there are multiple instances of the window manager.
 	tree="$(pstree -ps $$)"
