diff --git a/.local/bin/sysact b/.local/bin/sysact
index 451815d..f65f5fc 100755
--- a/.local/bin/sysact
+++ b/.local/bin/sysact
@@ -1,9 +1,6 @@
 #!/bin/sh
 # A dmenu wrapper script for system functions.
-case "$(readlink -f /sbin/init)" in
-	*systemd*) ctl='systemctl' ;;
-	*) ctl='loginctl' ;;
-esac
+ctl='systemctl'
 
 case "$(printf "🔒 lock\n🚪 leave dwm\n♻️ renew dwm\n🐻 hibernate\n💤 sleep\n🔃 reboot\n🖥️shutdown\n📺 display off" | dmenu -i -p 'Action: ')" in
 	'🔒 lock') slock ;;
